apps/flame-defi/app/route-animation.tsx

```diff
@@ -0,0 +1,19 @@
+"use client";
```

Comment:
```suggestion
"use client";

```
nit, but newlnie after use client statement